pkg/hintrunner/hint.go

```diff
@@ -87,3 +88,155 @@ func (hint TestLessThan) Execute(vm *VM.VirtualMachine) error {
 
 	return nil
 }
+
+type DivMod struct {
+	lhs       ResOperander
+	rhs       ResOperander
+	quotient  CellRefer
+	remainder CellRefer
+}
+
+func (hint DivMod) Execute(vm *VM.VirtualMachine) error {
+
+}
+
+type Uint256DivMod struct {
+	Dividend0  ResOperander
+	Dividend1  ResOperander
+	Divisor0   ResOperander
+	Divisor1   ResOperander
+	Quotient0  CellRefer
+	Quotient1  CellRefer
+	Remainder0 CellRefer
+	Remainder1 CellRefer
+}
+
+func (hint Uint256DivMod) String() string {
+	return "Uint256DivMod"
+}
+
+func (hint Uint256DivMod) Execute(vm *VM.VirtualMachine) error {
+
+	u128MaxFelt := MaxU128Felt()
+
+	dividend0, err := hint.Dividend0.Resolve(vm)
+	if err != nil {
+		return fmt.Errorf("resolve dividend0 operand %s: %v", hint.Dividend0, err)
+	}
```

Comment:
@rodrigo-pino replace %v with %w for err here too? is that a standard convention?

Reply:
Yes, at least we are following it because %w serves specifically for wrapping errors. %v gives you the default variable valuable. Is think %v is better used when you are not sure with what data type you're dealing with

Reply:
Was tweaking that part just know and I've noticed that we are using %v originally. That is a mistake on my part and I am already fixing it.